app/supplejack/transformation/field_execution.rb

```diff
@@ -15,16 +15,16 @@ def initialize(field)
     # rubocop:disable Security/Eval
     # rubocop:disable Lint/RescueException
     def execute(api_record)
-      WebMock.enable! unless Rails.env.test?
+      # WebMock.enable! unless Rails.env.test?
```

Comment:
do we want to keep this comment?

Reply:
Yeah this broke unfortunately like you thought it might :(.

Reply:
:'(